Maestro/subscriptions.json

```diff
@@ -1178,15 +1178,64 @@
         }
       }
     },
-    // Automate opening PRs to merge aspnet 3.0 branches back to master
+    // Automate opening PRs to merge aspnet 3.0 preview branches back to 3.0 stable branches
     {
       "triggerPaths": [
-        "https://github.com/aspnet/AspNetCore/blob/release/3.0-preview7/**/*",
-        "https://github.com/aspnet/AspNetCore-Tooling/blob/release/3.0-preview7/**/*",
-        "https://github.com/aspnet/EntityFramework6/blob/release/6.3-preview7/**/*",
-        "https://github.com/aspnet/EntityFrameworkCore/blob/release/3.0-preview7/**/*",   
-        "https://github.com/aspnet/Extensions/blob/release/3.0-preview7/**/*",
-        "https://github.com/aspnet/Blazor/blob/release/0.10.0-preview7/**/*",
+        "https://github.com/aspnet/AspNetCore/blob/release/3.0-preview8/**/*",
+        "https://github.com/aspnet/AspNetCore-Tooling/blob/release/3.0-preview8/**/*",
+        "https://github.com/aspnet/EntityFrameworkCore/blob/release/3.0-preview8/**/*",   
+        "https://github.com/aspnet/Extensions/blob/release/3.0-preview8/**/*",
+      ],
+      "action": "github-dnceng-branch-merge-pr-generator",
+      "actionArguments": {
+        "vsoSourceBranch": "release/3.0",
+        "vsoBuildParameters": {
+          "GithubRepoOwner": "release/3.0",
+          "GithubRepoName": "<trigger-repo>",
+          "HeadBranch": "<trigger-branch>",
+          "BaseBranch": "master"
```

Comment:
There's a lot of branch names here. Can you clarify exactly which branches mean which thing? Looking at `BaseBranch` seems to indicate this is going to create a PR to merge `release/3.0-preview8` to `master`, is that correct?

Reply:
This looks better now though I'm not sure what `vsoSourceBranch` is used for nor why it's 'master' regardless of where we're merging from and to.

Reply:
Found the answer in the [README.md](https://github.com/dotnet/versions/blob/master/Maestro/README.md) file in this folder:
> The source branch for which to queue the build.

I doubt AzDO pays attention to this anymore, given the bot's history of queuing "release/*" merge validation builds _correctly_ e.g. for our 'release/2.1' ➡️ 'release/2.2' merges.

Reply:
It's probably the branch of Arcade to use the script from. It might be fine to use the default (omit it) to use master, would have to check the build def to check.

Reply:
:ok: